docs/components/motor/_index.md

```diff
@@ -121,8 +68,16 @@ You can also ask questions on the [Viam Community Slack](https://join.slack.com/
 
 ## Next steps
 
+For general configuration and development info, see:
+
 {{< cards >}}
-{{% card link="/tutorials/get-started/confetti-bot/" %}}
-{{% card link="/tutorials/get-started/lazy-susan/" %}}
-{{% card link="/tutorials/configure/configure-rover" %}}
+{{% card link="/how-tos/configure/" noimage="true" %}}
+{{% card link="/how-tos/develop-app/" noimage="true" %}}
+{{% card link="/get-started/control-motor/" noimage="true" %}}
 {{< /cards >}}
+
+You can also use the motor component with the following services:
+
+- [motion service](/services/slam/): to move machines or components of machines
```

Comment:
```suggestion
- [motion service](/services/motion/): to move machines or components of machines
```
Really though this seems like a bit of a stretch, since you'd use arm or base with motion but not really motor directly AFAIK